test/test_ft_plot_cloud.m

```diff
@@ -75,6 +76,9 @@
 figure; ft_plot_cloud(pos, val, 'mesh', {mesh1, mesh2}, 'facealpha', 0, 'edgealpha', [.5 1], 'edgecolor', {'r', 'b'});
 figure; ft_plot_cloud(pos, val, 'mesh', {mesh1, mesh2}, 'facealpha', 0, 'edgealpha', [.5 1], 'edgecolor', {[1 0 0], [0 0 1]});
 
+% plot electrodes as discs
```

Comment:
why do you want to plot electrodes with ft_plot_cloud and not with ft_plot_sens?

Does this make the electrodes translucent or so?

Reply:
ft_plot_cloud is a helper function for ft_sourceplot and plots the electrodes as various shapes including clouds, discs, etc and color-codes those shapes based on the functional data.

Reply:
the difference between ft_plot_cloud and ft_plot_sens should remain clear. I don't want confusing/conflicting plotting implementations in both of  them. Clouds (like puffs of smoke) don't have a direction, electrodes do.

Reply:
plotting scalar values per channel, linked to the geometrical representation of that channel (e.g. as a disk for EEG, cylinder for sEEG, square for neuromag MEG), should be implemented on a single place.

3D rendering of geometry (cortex, scalp) together with scalar values on specific locations (e.g. at the electrodes or magnetometers) is a relevant feature to have, but should be designed in a wider perspective than just iEEG.

Reply:
If I am interpreting the above correctly, do you think we need a separate ft_plot_disc functionality? In a way, 'disc' is an elecshape so it could belong in ft_plot_sens, which already supports headshapes as inputs. It belongs less in ft_plot_cloud given that the discs do have a direction (being aligned with the headshape). However, ft_plot_cloud is the closest we got to plotting functional data in electrodes/clouds/etc -shape.